visualization/app/codeCharta/ui/codeMap/codeMap.render.service.ts

```diff
@@ -89,14 +91,19 @@ export class CodeMapRenderService {
 							showNodeName: showLabelNodeName,
 							showNodeMetric: showLabelNodeMetric
 						},
-						hightestNode
+						highestNode
 					)
 					amountOfTopLabels -= 1
 				}
 			}
 		}
 	}
 
+	private getHighestNodeDelta(sortedNodes: Node[]) {
+		const sortedNodeValues = sortedNodes.map(node => node.height + Math.abs(node.heightDelta))
+		return Math.max(...sortedNodeValues)
+	}
+
 	private getHighestNode(sortedNodes: Node[]) {
 		const sortedNodeValues = sortedNodes.map(node => node.height)
```

Comment:
```suggestion
	private getHighestNode(sortedNodes: Node[]) {
		const sortedNodeValues = sortedNodes.map(({ height, heightDelta }) => height + Math.abs(heightDelta ?? 0))
```

AFAIK the delta is only applied while in delta mode. Thus, it's simpler to just always add the value.